spec/concepts/user/find_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe User::Find do
+  it "finds a user guest" do
+    todo = Todo::Create.(todo: {title: "Testing helps!"}).model
+    expect(todo.list.user.fullname).to eq("Guest")
+  end
```

Comment:
Can you check if you have forgotten to implement User::Find spec? We're currently testing Todo::Create inside User::Find spec.